src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Form/suppliers_form_theme.html.twig

```diff
@@ -0,0 +1,75 @@
+{# **
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * #}
+
```

Comment:
Hoho, so we're providing a custom form theme block implementation for product page? 😄 

I like it, but we'll need to be extra explicit about it 😄  not the easiest thing to understand

Reply:
This one is not for the whole product page, it's only assigned to the suppliers sub form This is a classic way to use Symfony twig override system, nothing to document further I'd say 🤔

Reply:
Even if we do know Symfony and Twig well, not every PrestaShop contributors and users do 😉  .

No need to duplicate Symfony documentation, just need to say "and this `suppliers_form_theme.html.twig` file is actually an extension of PrestaShop default twig form theme, you can learn more about it from [this link]" :wink